src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,12 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a block in a Person's address in the address book.
```

Comment:
No need to mention it belongs to address book. This can be used in any address book _like_ application, no?